.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "3.6"
+install:
+  - pip install -r requirements.txt
+script:
+  - pyflakes assigner
+  - pylint assigner
```

Comment:
Let's leave `pylint` commented out until I fix all the remaining warnings! I'll start working on that but it'll be a bit (got some 5201 stuff to do first).

Reply:
Already did it, sorry. It's soothing as hell.

Reply:
You're a madman!